### PR TITLE
Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-docker.git`](https://github.com/cisagov/skeleton-docker.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

------------

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -155,6 +155,17 @@ Hub for a list of all the supported tags.
...
+## Ports ##
+
+The following ports are exposed by this container:
+
+| Port | Purpose        |
+|------|----------------|
+| 8080 | Example only; nothing is actually listening on the port |
+
+The sample [Docker composition](docker-compose.yml) publishes the
+exposed port at 8080.
```
Comment (jsf9k) [inline README.md:167]: This "Ports" section should be updated for this particular Docker container.

Any similar sections ("Volumes", etc.) should be updated as well, if they haven't already.

**Review by jsf9k — CHANGES_REQUESTED** [review]
I requested some minor changes.  Could someone who is familiar with this repo address my request?

**Comment by dav3r** [comment]
> I requested some minor changes. Could someone who is familiar with this repo address my request?

This repo has not had it's `first-commits` PR yet, so it is still essentially just the skeleton- for context, see [this comment](https://github.com/cisagov/pca-report-library/pull/4#issuecomment-852437603).  @ameliav - do you have any idea when you or someone on your team will be able to get this repo up and running (see #2)?

**Comment by ameliav** [comment]
@dav3r yes, I asked Ben and he said we should be working on other priorities for now, and get back to this part (taking out the docker parts of pca-report-library and moving them here) in 1Q22. And that's correct- I was following those steps when I created this repo which is just a copy of docker-skeleton, nothing more yet. Looks like the PR requests were auto-generated.

File: README.md (hunk 1 above)
Comment (mcdonnnj) [reply README.md:167]: This appears to be a placeholder repo right now. Once a `first-commits` branch is PRed this information will change as part of that.

Comment (dav3r) [reply README.md:167]: @mcdonnnj - you are correct on both counts.

Comment (jsf9k) [reply README.md:167]: Got it.  Thanks gents!

**Review by jsf9k — APPROVED** [review]

**Review by dav3r — APPROVED** [review]
👍

**Pull request merged**